apps/css-workshop/src/pages/icon.astro

```diff
@@ -53,74 +51,40 @@ import Layout from './_layout.astro';
 
   <h2>Icons with padding</h2>
   <section id='demo-padded'>
-    <svg-imodel-hollow
-      class='iui-svg-icon'
-      data-iui-icon-size='s'
-      data-iui-padded='true'
-      aria-hidden='true'></svg-imodel-hollow>
-    <svg-imodel-hollow
-      class='iui-svg-icon'
-      data-iui-icon-size='m'
-      data-iui-padded='true'
-      aria-hidden='true'></svg-imodel-hollow>
-    <svg-imodel-hollow
-      class='iui-svg-icon'
-      data-iui-icon-size='l'
-      data-iui-padded='true'
-      aria-hidden='true'></svg-imodel-hollow>
+    <Icon_ svg='imodel-hollow' size='s' data-iui-padded='true' />
+    <Icon_ svg='imodel-hollow' size='m' data-iui-padded='true' />
+    <Icon_ svg='imodel-hollow' size='l' data-iui-padded='true' />
   </section>
 
   <h2>Icons with status</h2>
   <section id='demo-status'>
-    <svg-info-circular class='iui-svg-icon' data-iui-icon-color='informational' aria-hidden='true'
-    ></svg-info-circular>
+    <Icon_ svg='info-circular' data-iui-icon-color='informational' />
```

Comment:
The `data-iui-icon-color` attribute should be handled within `Icon` using a `fill` prop (similar to react)